src/cityreader/cityreader.py

```diff
@@ -62,10 +97,43 @@ def cityreader(cities=[]):
 # TODO Get latitude and longitude values from the user
 
 def cityreader_stretch(lat1, lon1, lat2, lon2, cities=[]):
+    
   # within will hold the cities that fall within the specified region
-  within = []
-  
-  # Go through each city and check to see if it falls within 
+    within = []
+
+    lat1 = float(lat1)
+
+    lon1 = float(lon1)
+
+    lat2 = float(lat2)
+
+    lon2 = float(lon2)
+
+    df = pd.read_csv('cities.csv')
+
+    if lat1 < lat2:
+        df = df[df['lat'].between(lat1, lat2)]
+    else:
+        df = df[df['lat'].between(lat2, lat1)]
+    if lon1 < lon2:
+        df = df[df['lng'].between(lon1, lon2)]
+    else:
+        df = df[df['lng'].between(lon2, lon1)]
```

Comment:
I think I'm following here. You start with the entire dataframe, reduce the dataframe into rows that fall between lat1 and lat2, then further reduce it into rows that fall between lon1 and lon2. 

That between method on the dataframe returns a series - not entirely sure what a series is, but it looks like a series of row numbers(?) and True/False values - to specify which rows you want back from the original dataframe.

Had to play around with it a bit, but I think I kinda-sorta get it, hah! 

But, yeah this is pretty cool! Nice job 🎉